### ISSUE TITLE
Changing Kerbal in Command pod

### ISSUE THREAD

**Issue by BakerNaut** [linuxgurugamer/KCT#34]
Hitting clear or clear all, then selecting add compresses the GUI which does not allow you to select a new Kerbal.

**Comment by BakerNaut** [comment]
Video of demo of the problem: https://www.reddit.com/r/KerbalSpaceProgram/comments/dtw0vt/kerbal_construction_time173_error_anyone_else/?utm_source=share&utm_medium=web2x

**Comment by BakerNaut** [comment]
Video to demo the problem:  https://www.reddit.com/r/KerbalSpaceProgram/comments/dtw0vt/kerbal_construction_time173_error_anyone_else/?utm_source=share&utm_medium=web2x

**Comment by linuxgurugamer** [comment]
Please try this with the latest, I can't reproduce it right now

**Comment by topsailor** [comment]
Experiencing the identical issue as BakerNaut reported. 
Whenever I click the 'Add' button on KCT launch window, GUI breaks and following messeage spams the console.

KSP ver. 1.7.3 w/ all of the DLC. 


`
[EXC 14:49:52.552] NullReferenceException: Object reference not set to an instance of an object
	KerbalConstructionTime.KCT_GUI.DrawCrewSelect (Int32 windowID)
	UnityEngine.GUILayout+LayoutedWindow.DoWindow (Int32 windowID)
	UnityEngine.GUI.CallWindowDelegate (UnityEngine.WindowFunction func, Int32 id, Int32 instanceID, UnityEngine.GUISkin _skin, Int32 forceRect, Single width, Single height, UnityEngine.GUIStyle style)
[ERR 14:49:52.553] GUI Error: You are pushing more GUIClips than you are popping. Make sure they are balanced)`

and my screenshot.
![image](https://user-images.githubusercontent.com/15981604/71234619-f560e780-233c-11ea-82d7-59bfce7b3db8.png)

and attaching my [ksp.log. ](https://1drv.ms/u/s!AnCGI7wKaBjLldUV2cJwJssg40oqdw?e=qKDjyi)

At that stage only ESC menu, BonVoyage menu, Scansat menu are accessible. 

It started to happen in my ongoing game save after I installed WASD Editor camera, KerboKatz Utility, KerbalKonstruct, CustomPrelaunchChecks, BetterTimeWarpContinued, Janitor'sCloset, and OSSNTR. But removing all of the mods in the list didn't make the issue go away.

Both BakerNaut(As seen from his short video) and I use JNSQ and FARc and EVE, but those mods were there for a long time and the issue happened rather recently so I ruled out those from the possible cause.

I'll look for more clues whenever I can.

If You need any other information to resolve this one, please let me know.

**Comment by topsailor** [comment]
Update on the issue.
I had some time to test out which mod causes this incovenience. 
As always, I gradually removed installed mods until the problem went away and then I ended up like this.

![minimalInMod](https://user-images.githubusercontent.com/15981604/71303427-e2f9b300-23fb-11ea-928c-ef6e2a2804e5.png)

Under this minimal configuration, the same error still occurred.
So I did some tests.
1. Same configuration, DX9 or DX11 : Doesn't matter. Problem persists.
2. Same configuration , 1.7.3 or 1.8.1(Updated mods) : Same as above.
2. Same configuration , clear plugindata and configs : Same as above.
3. Same configuration, new save file:  Not a single error.

So the problem, actually, seems to reside inside my save file. 
I suspect that the corruption prevented reading the crew list from the file and made the GUI broken.
Now I have to tinker how to deal with this lovely 780742 lines of code.

Anyway, Sorry to bother you. 
Thanks always for supporting KSP mod community.
Have a nice weekend.
